package.json

```diff
@@ -60,7 +60,7 @@
     "@rollup/plugin-babel": "^5.3.1",
     "@rollup/plugin-node-resolve": "^11.0.1",
     "@rollup/plugin-replace": "^4.0.0",
-    "@rollup/plugin-typescript": "^8.5.0",
+    "@rollup/plugin-typescript": "^11.1.6",
```

Comment:
The nested `@react-router/dev/config` import failed type checking during the Rollup build until this was upgraded.